site/layouts/SiteIndex.jsx

```diff
@@ -1,14 +1,45 @@
 import React from 'react';
-import { Markdown } from 'antwar-helpers';
 import Hero from './Hero';
 
-const Index = ({ page }) => (
-  <div>
-    { page.title === 'Antwar' ?
-      <Hero page={page} /> :
-        <h1>{page.title}</h1> }
-    <Markdown page={page} />
+import classes from './SiteIndex.scss';
+
+const SiteIndex = () => (
+  <div className={classes.siteIndex}>
+    <Hero />
+    <article className={classes.introduction}>
+      <div className={classes.container}>
+        <div className={classes.inform}>
+          <section>
```

Comment:
Yeah, that's a cool idea. One more reason to push all the markdown processing to a loader. Exporting a structure like that shouldn't be a problem. 👍